homeassistant/components/imeon_inverter/strings.json

```
@@ -143,44 +173,116 @@
       "temp_component_temperature": {
         "name": "Component temperature"
       },
-      "monitoring_building_consumption": {
-        "name": "Monitoring building consumption"
-      },
-      "monitoring_economy_factor": {
-        "name": "Monitoring economy factor"
-      },
-      "monitoring_grid_consumption": {
-        "name": "Monitoring grid consumption"
-      },
-      "monitoring_grid_injection": {
-        "name": "Monitoring grid injection"
-      },
-      "monitoring_grid_power_flow": {
-        "name": "Monitoring grid power flow"
+      "monitoring_self_produced": {
+        "name": "Self produced"
       },
       "monitoring_self_consumption": {
-        "name": "Monitoring self-consumption"
+        "name": "Self consumption"
       },
       "monitoring_self_sufficiency": {
-        "name": "Monitoring self-sufficiency"
-      },
-      "monitoring_solar_production": {
-        "name": "Monitoring solar production"
+        "name": "Self sufficiency"
       },
       "monitoring_minute_building_consumption": {
-        "name": "Monitoring building consumption (minute)"
+        "name": "Building consumption"
       },
       "monitoring_minute_grid_consumption": {
-        "name": "Monitoring grid consumption (minute)"
+        "name": "Grid consumption"
       },
       "monitoring_minute_grid_injection": {
-        "name": "Monitoring grid injection (minute)"
+        "name": "Grid injection"
       },
       "monitoring_minute_grid_power_flow": {
-        "name": "Monitoring grid power flow (minute)"
+        "name": "Grid power flow"
       },
       "monitoring_minute_solar_production": {
-        "name": "Monitoring solar production (minute)"
+        "name": "Solar production"
+      },
+      "timeline_type_msg": {
+        "name": "Timeline status",
+        "state": {
+          "com_lost": "Communication lost",
+          "com_ok": "Communiation Ok",
+          "warning_grid": "Warning grid",
+          "warning_ond": "Warning inverter",
+          "warning_soft": "Warning software",
+          "warning_pv": "Warning PV",
+          "warning_bat": "Warning battery",
+          "warning_cpu": "Warning CPU",
+          "warning_spe": "Warning special",
+          "error_grid": "Error grid",
+          "error_ond": "Error inverter",
+          "error_soft": "Error software",
+          "error_pv": "Error PV",
+          "error_bat": "Error battery",
+          "error_spe": "Error special",
+          "info_grid": "Info grid",
+          "info_ond": "Info inverter",
+          "info_soft": "Info software",
+          "info_pv": "Info PV",
+          "info_bat": "Info battery",
+          "info_cpu": "Info CPU",
+          "info_spe": "Info special",
+          "warning_unknown": "Unknown warning",
+          "warnings": "Warning",
+          "error_unknown": "Unknown error",
+          "errors": "Error",
+          "good_1": "Ok",
+          "good_2": "Ok",
+          "good_3": "Ok"
```

Can be replaced with a reference:

```suggestion
          "good_1": "[%key:component::binary_sensor::entity_component::problem::state::off%]",
          "good_2": "[%key:component::binary_sensor::entity_component::problem::state::off%]",
          "good_3": "[%key:component::binary_sensor::entity_component::problem::state::off%]"
```